src/pages/components/leadspace.mdx

```diff
@@ -70,23 +85,28 @@ For detailed guidelines around usage, also reference the [Lead space pattern pag
 
 There are two color themes to choose from when using Lead space:
 
-- Use `Dark (gray-100)` when the background image has many shadows, is dimly lit, or uses a dark color palette. The gradient overlay will render dark and the text will be white.
-- Use `Light (white)` when the background image is well-lit or brightly colored. The gradient overlay will render white and the text will be gray-100.
+### Dark theme
+
+Use `Dark (gray-100)` when the background image has many shadows, is dimly lit, or uses a dark color palette. The gradient overlay will render dark and the text will be white.
 
-![Dark theme](../../images/pattern/leadspace/LeadSpace_ColorThemes_Dark.jpg)
+![Dark theme](../../images/pattern/leadspace/leadspace_colortheme_dark_demo.jpg)
 
-<Caption>Dark</Caption>
+<Caption>Dark Lead space – Super – 640px height at xlg 1312px breakpoint</Caption>
 <br />
 
-![Light theme](../../images/pattern/leadspace/LeadSpace_ColorThemes_Light.jpg)
+### Light theme
+
+Use `Light (white)` when the background image is well-lit or brightly colored. The gradient overlay will render white and the text will be gray-100.
+
+![Light_theme](../../images/pattern/leadspace/leadspace_colortheme_light_demo.jpg)
 
-<Caption>Light</Caption>
+<Caption>Light Lead space – Super – 640px height at xlg 1312px breakpoint</Caption>
 
-## Left-aligned style options
+## Type styles
 
-### Type styles
+### Lead space left-aligned
 
```

Comment:
If we're going to have a hierarchy to our types dependent on specific values in other props (I would strongly suggest we try not to do this) then we need to represent that hierarchy in our documentation. left styles should be under left, and centered styles should be under centered above in my opinion. This information is kind of sitting out here on its own.

Reply:
Let me know how do you want to proceed with this one.

Reply:
Settled.